libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -46,7 +46,7 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @DisplayName: Loiter Horizontal Maximum Speed
     // @Description: Defines the maximum speed in cm/s which the aircraft will travel horizontally while in loiter mode
     // @Units: cm/s
-    // @Range: 0 2000
+    // @Range: 50 2000
```

Comment:
We might want to do a minimum of 20 to be in line with SPEED parameter. Also the WPNAV_LOITER_SPEED_MIN define is 100, but I think that could go down to 20 too?